### PR TITLE
Update operations for Azure AD administrator on MI.

### PR DESCRIPTION
**Pull request by SanjaMalesevic**
This is a breaking change for .NET client. It brings managed instance admin API into sync with server admin API.
It is generated based on PR https://github.com/Azure/azure-rest-api-specs/pull/7377

### CODE REVIEW

**Comment by SanjaMalesevic** [comment]
Currently, I didn't add anything into section "New features:" from file Microsoft.Azure.Management.Sql.csproj. I am not sure what to add in case of updating existing client.
I've searched last few release notes and only found new features here.

**Comment by isra-fel** [comment]
Hi @SanjaMalesevic , could you
- Add a link to the swagger spec review PR in the description
- Include the `.txt` file generated when you run `generate.ps1`[(details)](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec)
- Since you are making a breaking change, please update the major version

Thanks

**Comment by SanjaMalesevic** [comment]
Hi @isra-fel,

Here is the link to swagger spec PR https://github.com/Azure/azure-rest-api-specs/pull/7377.
I've already included .txt file into CR (eng/mgmt/mgmtmetadata/sql_resource-manager.txt)
And, about major version, should I updated current AssemblyVersion from 1.0.0.0 to 2.0.0.0 or to 1.1.0.0? According that version, I suppose that I need to change also AssemblyFileVersion which is currently 1.38.0.0.

Thanks,
Sanja

File: sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj [hunk 1]
```diff
@@ -7,14 +7,9 @@
...
-		<Version>1.37.0-preview</Version>
+		<Version>2.0.0-preview</Version>
 		<PackageTags>Microsoft Azure SQL Management;SQL;SQL Management;</PackageTags>
-		<PackageReleaseNotes>
-			<![CDATA[
-New features:
-- Added support for failover databases readable secondary.
-      ]]>
-		</PackageReleaseNotes>
+		<PackageReleaseNotes>Operations Get, Create, Update and Delete from ManagedInstanceAdministratorsOperations have a different signature. Parameter "administratorName" is omitted.</PackageReleaseNotes>
```
Comment (isra-fel) [inline sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj]: Please wrap release notes with `<![CDATA[` and `]]>`, like it used to be. Thanks

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj (hunk 1 above)
Comment (SanjaMalesevic) [reply sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj]: Done

**Review by isra-fel — APPROVED** [review]

**Pull request merged**